test/types/type_variables/deitz/part7/test_infer_class2.chpl

```diff
@@ -3,10 +3,10 @@ class C {
   var f : t;
 }
 
-var c = new C(f = 1);
+var c = new borrowed C(f = 1);
 
 writeln(c);
 
-var d : C(int) = new C(int, 12);
```

Comment:
Did this test previously leak?

Reply:
no, because it doesn't compile, it tests a compilation error